src/main/java/Assignment1.java

```diff
@@ -5,11 +5,11 @@ public static void main(String[] args) {
         double d = i;
 
         //double to int
-        double x =5;
+        double x =5d;
         int y = (int) x;
 
         //double to float
-        double a =1000;
+        double a =1000d;
         float b = (float) a;
 
         // float to double
```

Comment:
for double to float vice versa 
we can minimize the loc
like:-
//double to float
double a =1000d;
float b = (float) a;

// float to double
a=b;
---------
Can you do it for the remaining questions.